web/src/components/vis/VisLoadings.vue

```diff
@@ -98,52 +96,38 @@ export default {
         points,
       } = this.$props;
 
-      // Draw the data.
-      //
-      // Plot the vectors and points in the scatter plot.
+      // Plot the vectors as a scatter plot.
       const svg = select(this.$refs.svg);
-      const sel = svg.select('g.plot');
-      sel.selectAll('line')
-        .data(points)
-        .join(enter => enter.append('line')
-          .attr('x1', this.scaleX(0))
-          .attr('y1', this.scaleY(0))
-          .attr('x2', this.scaleX(0))
-          .attr('y2', this.scaleY(0))
-          .style('stroke', 'black')
-          .style('opacity', 0.0))
-        .transition()
-        .duration(this.duration)
-        .attr('x2', d => this.scaleX(d.x))
-        .attr('y2', d => this.scaleY(d.y))
-        .style('opacity', 1.0);
-
       const tooltip = select(this.$refs.tooltip);
       const coordFormat = format('.2f');
-      sel.selectAll('circle')
+      const radius = 5;
+      svg.select('g.plot')
+        .selectAll('circle')
         .data(points)
         .join(enter => enter.append('circle')
           .attr('cx', this.scaleX(0))
           .attr('cy', this.scaleY(0))
           .attr('r', 0)
+          .style('fill-opacity', 0.001)
+          .style('stroke', 'black')
           .on('mouseover', function mouseover(d) {
             select(this)
               .transition()
-              .duration(200)
-              .attr('r', 4);
+              .duration(500)
```

Comment:
Just curious, what prompted this change?  I checked it out and the half-second animation felt a bit long.  I changed it back to 200 and that feels much more crisp to me.  This may be a case for Judge Tomeck to weigh in on.

Reply:
Well the substantive thing was to reduce the mouseout animation from 500 to 200 ms; the longer transition felt way too "sticky". I had a vague idea that a longer mouseenter animation served us better somehow but:
1. I'm fine with changing both durations to 200ms, and
2. I agree with asking Judge @jtomeck to weigh in here.

Reply:
Actually let's just go with 200ms. It does feel better, and any arguments I could make about a slower half of the animation are too weak to really belabor.